variables.mk

```diff
@@ -74,6 +74,18 @@ ifeq ($(SUB_PROJECT),hwacha)
 	TB                ?= TestDriver
 	TOP               ?= ExampleRocketSystem
 endif
+# Stand-in firechip variables
```

Comment:
@zhemao What should these be to simulate `firechip` in Chipyard?

Reply:
We've resolved to comment this out temporarily.